### ISSUE TITLE
Using event.code instead of event.key locks out disabled users

### ISSUE THREAD

**Issue by elfsternberg** [mattburesh/bluesky-shortcuts#19]
Using event.code means that people with non-standard keyboards such as Dvorak, Coleman, or any number of specialty keyboards meant to be used by those with amputations or palsy don't have the keys where their assistive devices expect, preventing them from using the extension.

I'd have a PR for this, but I'm at work when I noticed it. Maybe when I get home, and figure out how to load an extension from build. (Haven't worked on extensions before; mostly just used Tampermonkey.)

**Comment by mattburesh** [comment]
Thanks for bringing this up @elfsternberg! I didn't know that about `event.code`. If you're able to put something together that'd be awesome, otherwise I can probably get to it tomorrow or this weekend.

**Comment by mattburesh** [comment]
This should be fixed in the [latest release (v1.0.6)](https://github.com/mattburesh/bluesky-shortcuts/releases/tag/v1.0.6). The [Firefox add-on](https://addons.mozilla.org/en-US/firefox/addon/bluesky-shortcuts/) is up to date, but it will take a couple days for the chrome web store to approve the latest version.

I'm going to close this, but if you notice it still isn't working right go ahead and reopen it.

**Comment by elfsternberg** [comment]
Thank you! (I was just about to send a PR, but... whatever. As long as it helps.) And looking at the PR, yeah, that's exactly the same change I was gonna make:
```
const key = event.key.length === 1 ? event.key.toLowerCase() : event.key;
```
